src/main/java/seedu/anilist/logic/commands/CommandsList.java

```diff
@@ -4,7 +4,8 @@
  * List of all commands currently supported.
  */
 public class CommandsList {
-    private static final String[] LIST_OF_COMMANDS = {"add", "delete", "help", "list", "update"};
+    private static final String[] LIST_OF_COMMANDS =
+        {"add", "delete", "help", "list", "update", "status", "rename", "genre", "find", "clear"};
```

Comment:
Should we standardize the order of which the commands come in?
Like, alphabetical, or some other order?

Reply:
That would also affect ug, so I would say ideally the list of commands here should also come in the same order as it is in the ug. Not that important though.